jobs/cron/templates/bin/setup_crontab.erb

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -exu
+
+CONFIG_DIR=/var/vcap/jobs/cron/config
+
+mkdir -p ${CONFIG_DIR}
+rm -f ${CONFIG_DIR}/entry-*
+
+read -r -d '' CRONTAB <<'EOF'
+<% p("cron.variables", {}).each do |k, v| %>
+<%= k %>=<%= v %>
+<% end %>
+EOF
+
+<% p("cron.entries", []).each do |entry| %>
+<% if entry.key?("script") %>
+ENTRY=$(mktemp ${CONFIG_DIR}/entry-XXXXXX)
```

Comment:
The sysadmin in me kind of wants to have people name their script in the spec, so that if someone looks at the crontab, it's less confusing about what job is running when